src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -28,13 +28,12 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->normalizeKeys(false)
                     ->prototype('array')
                         ->children()
-                            ->arrayNode('client')
+                            ->variableNode('client')
                                 ->info('All options for the Elastica client constructor')
                                 ->example([
                                     'host' => '%env(ELASTICSEARCH_HOST)%',
                                     'transport' => '@JoliCode\Elastically\Transport\HttpClientTransport',
                                 ])
-                                ->prototype('variable')->end()
```

Comment:
can you add `defaultValue([])`, and revert the diff in the Extension? Thanks

Reply:
Fixed :+1: